code/src/nuvla/job/actions/utils/resource_log_fetch.py

```diff
@@ -95,7 +95,7 @@ def fetch_log(self):
                 self.get_components_logs()))
 
     def fetch_resource_log(self):
-        self.log.info(f'Job started for {self.resource_log_id}.')
+        self.log.info(f'Job started for {self.resource_log_id}')
```

Comment:
This is a totally monir change to allow copy/paster of the resource_log ID without having to deal with the tailing period punctuation mark.